README.md

```diff
@@ -66,7 +66,7 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.3/install.html#
 | | `bbb_kurento_interfaces` | Specify the listening interfaces for kurento | `{{ [ansible_default_ipv4.interface, 'lo'] }}` | |
 | | `bbb_nodejs_version` | version of nodejs to be installed | `12.x` | |
 | | `bbb_system_locale` | the system locale to use | `en_US.UTF-8` | |
-| | `bbb_secret` | define the secret for bbb | `none` | `set this if you want to define the bbb-conf -secret. Otherwise the secret is generated by bbb` |
+| | `bbb_secret` | define the shared secret for bbb | `none` | set this if you want to define the bbb-conf -secret. Otherwise the secret is generated by bbb. Supported characters are [a-zA-z0-9] |
```

Comment:
Uppercase and monospace is probably easier to read

```suggestion
| | `bbb_secret` | define the shared secret for bbb | `none` | set this if you want to define the bbb-conf -secret. Otherwise the secret is generated by bbb. Supported characters are `[a-zA-Z0-9]` |
```